docs/validators/notaryv1.md

```diff
@@ -0,0 +1,292 @@
+This is done via `validators` in `helm/values.yaml`.
+The corresponding entry should look similar to the following (using the extracted public key as trust root):
+
+```yaml
+- name: customvalidator
+  type: notaryv1
+  host: notary.docker.io
+  trust_roots:
+  - name: default
+    key: |  # THE DESIRED PUBLIC KEY BELOW
+      -----BEGIN PUBLIC KEY-----
+      MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOXYta5TgdCwXTCnLU09W5T4M4r9f
+      QQrqJuADP6U7g5r9ICgPSmZuRHP/1AYUfOQW3baveKsT969EfELKj1lfCA==
+      -----END PUBLIC KEY-----
+```
+
+You also need to create a corresponding entry in the image policy via `policy` in `helm/values.yaml`, for example:
+
+```yaml
+- pattern: "docker.io/<REPOSITORY>/<IMAGE>:*"  # THE DESIRED REPOSITORY
+  validator: customvalidator
+```
+
+After installation, you are ready to verify your images against your public key:
+
+```bash
+helm install connaisseur helm --atomic --create-namespace --namespace connaisseur
+```
+
+Connaisseur now rejects all images from the given repository that have not been signed based on the provided public key.
+A quick guide for installation and testing is available in [getting started](../getting_started.md#deploy-connaisseur).
+It also provides a full step-by-step guide.
+
+### Understanding validation
+
+Using the simple pre-configuration shipped with Connaisseur, it is possible to test validation by deploying some pods:
+
+```bash
+kubectl run test-signed --image=docker.io/securesystemsengineering/testimage:signed
+> pod/test-signed created
+
+kubectl run test-unsigned --image=docker.io/securesystemsengineering/testimage:unsigned
+> Error from server: admission webhook "connaisseur-svc.connaisseur.svc" denied the request: Unable to find signed digest for image docker.io/securesystemsengineering/testimage:unsigned.
+# or in case of a signature with a different key
+> Error from server: admission webhook "connaisseur-svc.connaisseur.svc" denied the request: Failed to verify signature of trust data root.
+```
+
+How does Connaisseur validate these requests and convert the images with tags to digests?
+What happens in the background is that Connaisseur looks up trust data of the image in the `root`, `snapshot`, `timestamp` and `targets` files (in json format) by querying the API of the Notary server.
+Trust data syntax is validated against their known schemas and the files' signatures are validated against their respective public keys.
+The pinned root key is used for the `root.json` file that in turn contains the other keys which can then be trusted for validation of the remaining trust data (`snapshot.json`, `timestamp.json`, `targets.json`).
+Furthermore, Connaisseur gathers trust data of potential delegations linked in the `targets` file which can then be used to [enforce delegations](#enforcing-delegations).
+
+At this point, Connaisseur is left with a set of potentially disjoint or overlapping sets of trust data.
```

Comment:
I think this sentence is hard to understand. Before, we spoke about the trust data and now it's sets of trust data and it's unclear how these sets are derived.

Reply:
Maybe something along
```
At this point, Connaisseur is left with a set of potentially disjoint or overlapping sets of trust data (i.e. signatures on digests).
Connaisseur filters the trust data for signed digests that actually relate to the image under validation.
In case exactly one trusted digest remains, Connaisseur modifies the admission request and admits it.
Otherwise, admission is rejected (this could happen if Alice signs image:tag linking to image:shaabc and Bob signs image:tag linking to image:shadef.
```
?

Reply:
proposal